vscode-extension/src/aiConfigEditor.ts

```diff
@@ -199,17 +234,17 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
 
     // Make sure we get rid of the listener when our editor is closed.
     webviewPanel.onDidDispose(() => {
+      isWebviewDisposed = true;

```

Comment:
Ok so just to clarify, this needs to be moved up top here becasue it's a flag that will get read on L171? And L171 flow occurs when we call `changeDocumentSubscription` on L240 below?

Reply:
When testing it didn't actually seem to matter if this was at the top or bottom but it makes more sense to set it immediately, considering there are async events that are checking it